test/test_session_limitable.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class TestSessionLimitable < ActiveSupport::TestCase
```

Comment:
Based on your good point about missing test coverage - what do you think about adding tests for this module first in a separate PR and then refactoring?

Reply:
I'm trying to keep the refactors to a minimum, but I can pull them out entirely if you think it helps keep things cleaner.

Reply:
What you have here is fine to me. If this was going to be a long `WIP` branch that's when my suggestion comes in.

Reply:
👍 Once I finish up another subset of tests it should be good to go.

Reply:
Have the other tests been wrapped up?